### ISSUE TITLE
Is it weird to iron your bed sheets?

### ISSUE THREAD

**Issue by theresaanna** [khandelwal/AskShashank#3]
Cause I left mine in the dryer overnight and now they're like perma-wrinkled.

**Comment by cmc333333** [comment]
Won't laying on them straighten them out?

**Comment by theresaanna** [comment]
Only on the places I lay on. Also this includes duvet covers. Yes, coverS. We are very European.

**Comment by Scotchester** [comment]
Yes.

**Comment by micheletepper** [comment]
Washing and drying them again would be wasteful, but possibly easier.

**Comment by theresaanna** [comment]
I totally tried putting the dryer back on for a bit but those wrinkles are deep set.

**Comment by iuloshi** [comment]
Put them in the bathroom while you shower and try to steam the wrinkles out? This coming from the person to owns an ironing board, but not an iron. @ohsk can confirm. There were sad, wrinkled popovers in Brooklyn.

**Comment by hkeeler** [comment]
Have room in your apartment for one of [these](http://www.alibaba.com/product-detail/YP-steam-industrial-laundry-flatwork-ironer_1612163984.html?s=p)?

**Comment by theresaanna** [comment]
@hkeeler Obvs. I was wondering what we were going to use that eighth bedroom for. (Also, you live! And we're almost neighbors now!)

**Comment by theresaanna** [comment]
@iuloshi I wish I'd known that! I had two irons in Brooklyn. I would have given you one.

**Comment by efortiz** [comment]
@theresaanna I iron mine, and fold them, too - it's civilized. Don't let the haters discourage you!

**Comment by theresaanna** [comment]
Thank you, @efortiz!

**Comment by hkeeler** [comment]
@theresaanna, yes, I am alive.  Please don't tell anyone.  And welcome to Cali!

@efortiz, this does not surprise me.
